python/lsst/ts/hexrotcomm/command_telemetry_client.py

```diff
@@ -397,20 +378,37 @@ async def next_telemetry(self):
         await self._telemetry_task
         return self.telemetry
 
-    async def put_command(self, command):
-        """Write a command to the controller.
+    async def run_command(self, command, interrupt=False):
+        """Run a command and wait for acknowledgement.
 
         Parameters
         ----------
         command : `Command`
             Command to write. Its counter field will be set.
+        interrupt : `bool`, optional
+            Interrupt the current command, if any?
+            Only use this for the stop command and similar.
```

Comment:
I prefer not to document default values because it duplicates information, which can easily go out of date. Sphinx shows the default values in the call signature.